packages/ui-components/src/lib/components/tables/VaultsListTable.svelte

```diff
@@ -44,40 +44,111 @@
 	const { account, matchesAccount } = useAccount();
 	const raindexClient = useRaindexClient();
 
-	$: owners =
-		$activeAccountsItems && Object.values($activeAccountsItems).length > 0
-			? Object.values($activeAccountsItems)
-			: $showMyItemsOnly && $account
-				? [$account]
-				: [];
+	// Use our new filter store instead of props
+	const filterStore = useFilterStore();
+
+	// Get filters from our store
+	$: currentFilters = $filterStore?.getVaultsFilters() || {
+		owners: [],
+		hideZeroBalance: false,
+		tokens: undefined,
+		chainIds: undefined
+	};
+
+	// Derive values from filter store
+	$: selectedChainIds = currentFilters.chainIds || [];
+	$: hideZeroBalanceVaults = currentFilters.hideZeroBalance;
+	$: activeTokens = currentFilters.tokens || [];
+
+	// Create writable stores that sync with our filter store
+	// These will be used by ListViewOrderbookFilters for direct updates
+	const selectedChainIdsStore = writable<number[]>([]);
+	const hideZeroBalanceVaultsStore = writable<boolean>(false);
+	const activeTokensStore = writable<`0x${string}`[]>([]);
```

Comment:
i am not sure if these should be here. while porting the filters to the sdk we wanted to remove the filter stores from the apps but this just adds the filter logic on top of what we have. we should be able to manipulate filter store without needing to add extra state layer in the apps